pkgs/tools/misc/z/default.nix

```diff
@@ -0,0 +1,45 @@
+{ fetchFromGitHub, lib, stdenv }:
+  stdenv.mkDerivation
+    { pname = "z";
+      version = "1622481819";
```

Comment:
There's version 1.9? 
But if you insist on a non-release commit, then please use the `unstable-yyyy-mm-dd` format

Reply:
the 1.9 tag is from 2015, there have been updates since then, and there doesn't appear to be any attempt to version them.